docs/manual/layout/templates/manage-template.de.md

```diff
@@ -25,11 +25,38 @@ Template-Dateien in weiteren (unverknüpften) Unterordnern werden bei der Auswah
 ## Dateinamen
 
 Die Template-Dateien werden zur Erkennung mit einem Präfix versehen. Beispielsweise deutet `ce_` auf ein
-Inhaltselement (**c**ontent **e**lement) hin. Möchte man z.&nbsp;B. die Ausgabe des Inhaltselements vom Typ »Text«
-ändern, kann man das Template `ce_text.html5` hierzu verwenden. 
-
-In diesem Fall haben die Template-Änderungen Auswirkung auf alle Inhaltselemente vom Typ »Text«. Dies ist nicht immer
-erwünscht. Zur gezielten Nutzung kann man das Template individuell bezeichnen. Hierbei muss die jeweils vorgegebene
-Template-Bezeichnung beibehalten und lediglich erweitert werden: also z.&nbsp;B. `ce_text.html5` 
-umbenennen nach `ce_text_individuell.html5`. Dieses Template kann dann gezielt zur Ausgabe für ein (o. mehrere)
-Inhaltselement(e) vom Typ »Text« genutzt werden.
+Inhaltselement (**c**ontent **e**lement) hin. 
+
+
+{{< tabs groupId="templateGroup">}}
+{{% tab name="PHP" %}}
+
+
+Möchte man z.&nbsp;B. die Ausgabe des Inhaltselements vom Typ »Text« ändern, kann man das Template `ce_text.html5` 
+hierzu verwenden. In diesem Fall haben die Template-Änderungen Auswirkung auf alle Inhaltselemente vom Typ »Text«. 
+
+Dies ist nicht immer erwünscht. Zur gezielten Nutzung kann man das Template individuell bezeichnen. Hierbei muss die 
+jeweils vorgegebene Template-Bezeichnung beibehalten und lediglich erweitert werden: also z.&nbsp;B. `ce_text.html5` 
+umbenennen nach `ce_text_individuell.html5`. 
+
+Dieses Template kann dann gezielt zur Ausgabe für ein (oder mehrere) Inhaltselement(e) vom Typ »Text« genutzt werden.
+
+
+{{% /tab %}}
+{{% tab name="Twig" %}}
+
+
+{{< version "4.12" >}}
+
+Möchte man z.&nbsp;B. die Ausgabe des Inhaltselements vom Typ »Text« ändern, kann man das Template `ce_text.html.twig` 
+hierzu verwenden. In diesem Fall haben die Template-Änderungen Auswirkung auf alle Inhaltselemente vom Typ »Text«. 
+
+Dies ist nicht immer erwünscht. Zur gezielten Nutzung kann man das Template individuell bezeichnen. Hierbei muss die 
+jeweils vorgegebene Template-Bezeichnung beibehalten und lediglich erweitert werden: also z.&nbsp;B. `ce_text.html.twig` 
+umbenennen nach `ce_text_individuell.html.twig`. 
+
+Dieses Template kann dann gezielt zur Ausgabe für ein (o. mehrere) Inhaltselement(e) vom Typ »Text« genutzt werden.
```

Comment:
Zumindest inhaltlich ok? Wird ja für die finale 4.13 dann so möglich sein .

Frage ist dann ob dies auch für die 4.12 dann noch umgesetzt wird. 
Anderenfalls sollten wir hier wieder die version shortcodes auf 4.13 setzen ...?

Reply:
No, this will not be ready for the last 4.12 release, but it won't be in 4.13.0 either. :slightly_smiling_face:

> Anderenfalls sollten wir hier wieder die version shortcodes auf 4.13 setzen ...?

IMHO that's not necessary as 4.12 won't be maintained at that point and we could consider this a *bug*. But up to you.

Reply:
4.12 ist bei TWIG doch richtig, wir können doch nicht einzelne Funktionen Versionieren. Ausserdem wird 4.12 mit dem Erscheinen von 4.13 nicht mehr unterstützt. Ist halt blöd, wenn im Handbuch steht 4.13 und wir auf die Dev verlinken und da steht 4.12.

Reply:
> ...but it won't be in 4.13.0 either.

Oh, dann hab ich das komplett falsch verstanden. 
Wir wollen ja jetzt eh warten (s,o.), 
aber wenn dies auch nicht mit der 4.13 kommt ??

Reply:
Es kommt nicht in der 4.13.**0**, aber es kommt schon (mit an Sicherheit grenzender Wahrscheinlichkeit) in der 4.13.